jdaviz/core/loaders/resolvers/url/url.py

```diff
@@ -38,6 +40,13 @@ def is_valid(self):
 
     @observe('url', 'cache', 'timeout')
     def _on_url_changed(self, change):
+        # _uri_output_file is a string to the local path which uses the same file name
+        # as that from the URL. By stripping the local path, we can match to the URL.
+        # We check for bool for 'cache' on/off.
+        if (self._uri_output_file.lstrip(f"{self.local_path}") not in self.url.strip()
+                or isinstance(change['new'], bool)):
```

Comment:
does this work as well(and avoid calling the call/cache in the if-statement)?
```suggestion
        if change['name'] in ('url', 'cache'):
```

Reply:
I have a feeling the latter but let me check...

Reply:
Figured out my issue and fixed!

Reply:
FWIW, it wasn't re-downloaded, I was using `hasattr` to check for `_uri_output_file`; I didn't realize `hasattr` triggers the property even if it's cached, hence the double download I was seeing in discussing it here.